### ISSUE TITLE
OpenUlmus.org Prospectus and Donation Tiers (on OpenCollective)

### ISSUE THREAD

**Issue by cleverington** [elmsln/openulmus.org#43]
## Issue

We've been discussing and mostly finalized the 'Donation Tiers' and such over the last month (see https://github.com/elmsln/openulmus.org/wiki/5.-Budget-and-Sustainability).

As we're building a Prospectus in the next couple of weeks and putting the spit-shine on our content, we need to look at updating the Tiers as well.

I want to approach "this incorporated" (Phase 2, Amazee, etc.) and let them know that they'll get their money's worth if donating. However, they should also know what button and what _size_ button their logo will be.

Using the DrupalCon / DrupalCamp / etc. model, we can have something like this:

- Elite Tier
  - Logo on ReadMe and OpenCollective = 100x100
  - Logo on OpenUlmus = 120x120
- n00b Tier
  - 85x85
  - 100x100
- Basic Tier
  - 10x10
  - 10x10

**Comment by gusaus** [comment]
First off sponsor/backer tiers should display on our collective (https://opencollective.com/openulmus) and website (https://openulmus.org/). The following projects provide good examples of incorporating OpenCollective into their project/marketing sites.

https://webpack.js.org/
https://reactiveui.net/

By incorporating Polymer https://github.com/opencollective/opencollective-api/issues/769 into our MVP https://github.com/elmsln/openulmus.org/issues/1, we'll be able to have more control over display - also an even better platform we can provide as a service.

**Comment by gusaus** [comment]
With regards to donation tiers, there are a couple others we should be able to scope out and create with relative ease. 

For instance, we could offer something like the following once we determine the baseline price for white label Open Ulmus https://github.com/elmsln/openulmus.org/issues/21

## Platform sponsorship
Platform sponsorship is for organizations interested sponsoring development of our hosted Open Ulmus platform. We have either have tiers on https://opencollective.com/openulmus or stand-alone collective for Open Ulmus platform. The latter would be a way to decouple the #OSS project and the service offering.

### Ten Hours Custom Setup
Logo on Open Ulmus website
Ten hours custom setup and training
$1000 USD

### Twenty Hours Custom Setup
Logo on Open Ulmus website
Twenty hours custom setup and training
$1500 USD

### One Year Hosting
Logo on Open Ulmus website
One year free on our hosted platform (when ready)
$750 USD

### One Year Hosting & Custom Setup
Logo on Open Ulmus website
One year free on our hosted platform (when ready)
Ten hours custom setup and training
$1500 USD

**Comment by mmilutinovic013** [comment]
Is each of these sections for one-time payments?

**Comment by gusaus** [comment]
Here's a gdoc that includes the existing copy and tiers on https://opencollective.com/openulmus and the additional tiers we're working out in this issue - 
https://docs.google.com/document/d/11Wz3Qj1IVZHkbDzi0mi4eee4r-ulkh3Qs6pmHB5RIoY/edit?usp=sharing

I'd like to see if we can work to solidify some of the additional tiers and lock down how to display on the primary domain https://github.com/elmsln/openulmus.org/issues/34 asap.

**Comment by mmilutinovic013** [comment]
I added Ninja tier instead of N00b tier. I'm content with the ranges for each of these. Hosting costs, in general, I'll estimate in #21.

**Comment by mmilutinovic013** [comment]
Additionally, do you see benefit @gusaus, @cleverington, @kat-wehr in having a tier where we have @kat-wehr or other team members advise on how to make training for a certain amount of hours. let's say 20 hours of training creation training for $1500?

**Comment by gusaus** [comment]
Definitely! Probably the best would be like a certain amt of hours per month to work on whatever is determined priority. There are other collectives trying our similar models. Might not be a bad idea to talk out options in the #opensource room.
